### DISCUSSION TITLE
ALARM 8011  MotoROS2: Fatal Error [5]

### DISCUSSION THREAD

**Discussion by KanishkNavale** [Yaskawa-Global/motoros2#153] [Troubleshooting] [answered]
Hello,

Even tough we follow the setup instructions carefully, we get the following error:

```
ALARM 8011
 MotoROS2: Fatal Error
[5]
```

We have checked all the steps repeatedly and not found any consistency. I am attaching the following details,
Controller: YCR1000
ROS: Humble
[CMOS.zip](https://github.com/Yaskawa-Global/motoros2/files/12619008/CMOS.zip)
[PANELBOX.LOG](https://github.com/Yaskawa-Global/motoros2/files/12619002/PANELBOX.LOG)

**Comment by ted-miller** [comment]
Hi @KanishkNavale 
Can you please run the [debug listener](https://github.com/Yaskawa-Global/motoros2/blob/main/doc/troubleshooting.md#debug-log-client) script while your controller boots up? This will show any output from the controller and hopefully explain why the error occurred. Also, a copy of the yaml configuration file would be helpful.

Please note that there have been two recent occurrences of this same alarm, caused by this. https://github.com/Yaskawa-Global/motoros2/issues/144
The solution in those cases was to just override the node name in your yaml file. 

--------------------------------
EDIT: There will be a fix for #144 in the next released version.

**Comment by KanishkNavale** [comment]
Hi @ted-miller ,
[motoros2_config.zip](https://github.com/Yaskawa-Global/motoros2/files/12619592/motoros2_config.zip)


Thank you for replying as soon as possible. I have attached the yaml config file.

**Comment by ted-miller** [comment, marked as answer]
Try changing the default node name like so:

![image](https://github.com/Yaskawa-Global/motoros2/assets/4623811/2c031451-3098-4ceb-b152-ac54acd7c5e6)

If that doesn't work, we'll need to see the output from the debug-listener script.

**Comment by KanishkNavale** [comment]
@ted-miller, do you want me to update the yaml file and create a PR?

**Reply by gavanderhoorn** [reply]
Thank you for the offer, but that is not needed.

As @ted-miller mentioned in https://github.com/Yaskawa-Global/motoros2/discussions/153#discussioncomment-7015351, the issue you most likely encountered has been fixed (#145) and this fix will be included in the next release of MotoROS2 (`0.1.2`).
